web/modeshape-web-explorer/src/main/java/org/modeshape/web/server/JcrServiceImpl.java

```
@@ -352,15 +352,16 @@ private AccessControlList findAccessList( AccessControlManager acm,
             String name = def.getName();
             String type = PropertyType.nameFromValue(def.getRequiredType());
             
-            Property p;
+            Property p = null;
             try {
                 p = node.getProperty(def.getName());
-                String display = values(def, p);
-                String value = p.isMultiple() ? multiValue(p)
-                        : singleValue(p, def, repository, workspace, path);
-                list.add(new JcrProperty(name, type, value, display));
             } catch (PathNotFoundException e) {
```

should this exception be ignored ?